Cargo.toml

```diff
@@ -14,7 +14,7 @@ name = "egui_tiles"
 readme = "README.md"
 repository = "https://github.com/rerun-io/egui_tiles"
 rust-version = "1.81"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
This is a bit weird - we usually do this as late as possible, i.e. during the release:

https://github.com/rerun-io/egui_tiles/blob/main/RELEASES.md